src/ibex_bluesky_core/callbacks/file_logger.py

```diff
@@ -59,10 +60,19 @@ def start(self, doc: RunStart) -> None:
         title_format_datetime = datetime_obj.astimezone(ZoneInfo("Europe/London")).strftime(
             "%Y-%m-%d_%H-%M-%S"
         )
-        axes = "_".join(self.fields)
         rb_num = doc.get("rb_number", "Unknown RB")
+
+        detectors = doc.get("detectors", [])
+        # motors is a tuple, we need to convert to a list to join the two below
+        motors = doc.get("motors", [])
+        if motors:
+            motors = list(motors)
+
+        detectors_and_motors = "_".join(detectors + motors)
```

Comment:
My interpretation of https://github.com/ISISComputingGroup/ibex_bluesky_core/issues/24#issuecomment-2590275747 was that `instrument_axis_date_time` means `axis` is just the motors - explicitly _not_ all the detectors (of which there may be very many, whereas in pretty much all cases there'll only be one or _maybe_ a few motors.

i.e. for a scan of `theta` against `dae-intensity` and `dae-intensity_err` I think the filename should be:

```
CRISP_theta_2024_01_01-01_01_01Z.csv
```